routes/index.js

```diff
@@ -238,7 +238,7 @@ exports.create = function (req, res, next) {
 };
 
 // Insert new vulnerable code:
-/*
+
 exports.destroy = function (req, res, next) {
   Todo.findById(req.params.id, function (err, todo) {
```

Comment:
### <sub><img src="https://res.cloudinary.com/snyk/image/upload/r-d/scm-platform/snyk-pull-requests/severity-high.svg" width="20"/></sub>&nbsp; NoSQL Injection

Unsanitized input from `an HTTP parameter` flows into `findById`, where it is used in an NoSQL query. This may result in an NoSQL Injection vulnerability.

<h6>
<a href="https://cwe.mitre.org/data/definitions/943.html">CWE-943</a> | <a href="https://docs.snyk.io/manage-risk/prioritize-issues-for-fixing/priority-score">Priority score 900</a> | Line 243 | <a href="https://learn.snyk.io/lesson/nosql-injection-attack/">Learn more about this vulnerability</h6>

<details>

<summary><b>Data flow: 5 steps</b></summary>

`Step 1` https://github.com/DefenderK/nodejs-goof/blob/ae8278b4f0a7333001b392526d51d98ccc47727a/routes/index.js#L243

`Step 2` [routes/index.js#243](https://github.com/DefenderK/nodejs-goof/blob/ae8278b4f0a7333001b392526d51d98ccc47727a/routes/index.js#L243)

`Step 3` [routes/index.js#243](https://github.com/DefenderK/nodejs-goof/blob/ae8278b4f0a7333001b392526d51d98ccc47727a/routes/index.js#L243)

`Step 4` [routes/index.js#243](https://github.com/DefenderK/nodejs-goof/blob/ae8278b4f0a7333001b392526d51d98ccc47727a/routes/index.js#L243)

`Step 5` https://github.com/DefenderK/nodejs-goof/blob/ae8278b4f0a7333001b392526d51d98ccc47727a/routes/index.js#L243

</details>